dashboard/test/ui/runner.rb

```diff
@@ -98,6 +98,11 @@
 $failures = []
 
 if $options.local
+  #Verify that chromedriver is actually running
+  unless `ps`.include?('chromedriver')
+    puts 'You cannot run with the --local flag unless you are running chromedriver. Please run it'
+    exit(1)
+  end
```

Comment:
Should we even try running it for them, and then only exit if we fail to find it or it fails to execute for some reason?